### DISCUSSION TITLE
Issue with Building all Apps in Next.js 14 within a Monorepo Setup

### DISCUSSION THREAD

**Discussion by Muhammad-Hashim** [vercel/turborepo#7782] [Help] [answered]
### Summary

I am encountering an error while building all apps in Next.js 14 within my monorepo setup. Here's the breakdown of the situation:

Monorepo Structure: I have organized my project into a monorepo, consisting of multiple applications including:

App 1: Web application
App 2: Agent portal
App 3: Superadmin
Other packages
Technologies Used:

Next.js 14.1.0 is employed in the web application (App 1).
Agent portal (App 2) and Superadmin (App 3) are currently on version 13.4 of Next.js.
Error Description:
Despite attempting to build the agent portal with Next.js 13 as specified in its package.json, the build process consistently targets Next.js 14. This inconsistency persists across all apps, as the build always proceeds with Next.js 14, disregarding the specified version.

This issue disrupts the desired version control within the monorepo setup, impeding the ability to maintain separate Next.js versions for different applications.

Your assistance in resolving this discrepancy and ensuring accurate versioning during the build process would be greatly appreciated.

**Comment by anthonyshew** [comment, marked as answer]
This is the responsibility of your package manager, not Turborepo. Depending on which package manager you're using, your settings for it, and other factors, dependency resolving could be happening a number of ways.
